src/pudl/glue/xbrl_dbf_ferc1.py

```diff
@@ -0,0 +1,210 @@
+"""Module to help update the id's associated with utilities."""
```

Comment:
this whole module is a major WIP.

The only thing that is being employed in here right now are `get_util_ids_ferc1_csv` and `get_util_ids_pudl`.

The rest was the start of being able to automatically(-ish) update these ID's with new data. I decided to put a pause on that after coming up with at least a working idea of how to do it with the current file setup in order to shift to actually getting these IDs fed into the db.